vitirover:
  # Joint State Controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Velocity Controllers for Wheels
  left_front_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: front_left_wheel_joint
    pid: {p: 1.0, i: 0.0, d: 0.0}

  right_front_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: front_right_wheel_joint
    pid: {p: 1.0, i: 0.0, d: 0.0}

  left_rear_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: rear_left_wheel_axle_joint
    pid: {p: 1.0, i: 0.0, d: 0.0}

  right_rear_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: rear_right_wheel_axle_joint
    pid: {p: 1.0, i: 0.0, d: 0.0}

  # Position Controller for Rear Axle
  rear_axle_position_controller:
    type: effort_controllers/JointPositionController
    joint: rear_axle_joint
    pid: {p: 1.0, i: 0.0, d: 0.0}
